### ISSUE TITLE
Cannot reproduce the paper result

### ISSUE THREAD

**Issue by xiuzbl** [zxlzr/FewShotNLP#8]
First, I cannot reproduce the result (90% accuracy) which is reported in the paper based on your code and your method: pretrained bert + unsupervised fine-tuning + MAML.

Second, I noticed that there are overlaps between meta-training and meta-testing datasets, which means your meta-training also contain meta-testing domains. I think this is a problem for your code implementation.

**Comment by zxlzr** [comment]
Hi, sorry for the late reply and I notice your issue, the 90% accuracy is obtained by roberta （https://github.com/zxlzr/FewShotNLP/issues/4#issue-601735452）rather than pretrained bert, sorry for the missing details of the pre-trained language models. 

We follow previous studies (Diverse few-shot text classification with multiple metrics) to split the meta-training and meta-testing, this is a good fining and we will try to fix it.
